src/style.css

```diff
@@ -1 +1,69 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.page__body {
+  margin-top: 0;
+}
+
+.search-bar--big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search-bar__input {
```

Comment:
according to the mockup, there should be a gradient background on the focus. You can add 2: image and gradient as a background - google how to do it properly. Or add image using tag